### DISCUSSION TITLE
help~qwq

### DISCUSSION THREAD

**Discussion by KarasKim** [NateScarlet/auto-derby#378] [Q&A]
一开始是error: can't find Rust compiler
后面看到说是要执行一下：py -3.8 -m pip install --upgrade pip
完成之后再执行py -3.8 -m pip install -r requirements.txt
就变成这样哩（我安装的64位的）
![B8G(DGC{HGACJSA54N74X8U](https://github.com/NateScarlet/auto-derby/assets/153373757/488632bb-007c-48b8-8435-6dcc1ad927c4)

**Comment by NateScarlet** [comment]
你系统设了代理 但是代理连不上

**Reply by KarasKim** [reply]
okok。
